packages/request-manager/e2e/interceptor.test.ts

```diff
@@ -44,8 +45,9 @@ describe('Interceptor', () => {
             port,
             controlPort,
             torDataDir,
+            snowflakeBinaryPath,
         });
-        const torParams = torController.getTorConfiguration(processId);
+        const torParams = await torController.getTorConfiguration(processId, false);
```

Comment:
The second param probably shouldn't be bool anymore.

Reply:
https://github.com/trezor/trezor-suite/pull/12991/commits/c53bdc9fcf78fcf83c266fad4f8d5792eb01e73e